example-app/app.json

```diff
@@ -12,20 +12,24 @@
     },
     "plugins": [
       [
-        "@stripe/stripe-terminal-react-native",
+        "../lib/commonjs/plugin/withStripeTerminal",
```

Comment:
our example-app uses the publicly released SDK, i don't think we should reference a local development plugin